### ISSUE TITLE
--new-pr bails out on github 307 redirect

### ISSUE THREAD

**Issue by zao** [easybuilders/easybuild-framework#2257]
After the move of upstream repositories to the "easybuilders" account, --new-pr bails out after pushing the branch to the fork, assumedly when it's going to make the PR.

```
[zao@freja GCCcore]$ eb --new-pr GCCcore-6.3.0.eb
== temporary log file in case of crash /eb/tmp/eb-EUvLu4/easybuild-cIS352.log
== fetching branch 'develop' from https://github.com/hpcugent/easybuild-easyconfigs.git...
== copying easyconfigs to /eb/tmp/eb-EUvLu4/git-working-dirXd1UQ7/easybuild-easyconfigs...

Opening pull request
* target: hpcugent/easybuild-easyconfigs:develop
* from: zao/easybuild-easyconfigs:20170705094024_new_pr_GCCcore630
* title: "{compiler}[dummy/] GCCcore v6.3.0"
* description:
"""
(created using `eb --new-pr`)

"""
* overview of changes:
 easybuild/easyconfigs/g/GCCcore/GCCcore-6.3.0.eb | 1 +
 1 file changed, 1 insertion(+)

Traceback (most recent call last):
  File "/usr/lib64/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/eb/software/EasyBuild/3.3.0/lib/python2.7/site-packages/easybuild_framework-3.3.0-py2.7.egg/easybuild/main.py", line 478, in <module>
    main()
  File "/eb/software/EasyBuild/3.3.0/lib/python2.7/site-packages/easybuild_framework-3.3.0-py2.7.egg/easybuild/main.py", line 401, in main
    commit_msg=options.pr_commit_msg)
  File "/eb/software/EasyBuild/3.3.0/lib/python2.7/site-packages/easybuild_framework-3.3.0-py2.7.egg/easybuild/tools/github.py", line 926, in new_pr
    status, data = pulls_url.post(body=body)
  File "/eb/software/EasyBuild/3.3.0/lib/python2.7/site-packages/vsc_base-2.5.8-py2.7.egg/vsc/utils/rest.py", line 144, in post
    return self.request(self.POST, url, json.dumps(body), headers, content_type='application/json')
  File "/eb/software/EasyBuild/3.3.0/lib/python2.7/site-packages/vsc_base-2.5.8-py2.7.egg/vsc/utils/rest.py", line 175, in request
    conn = self.get_connection(method, url, body, headers)
  File "/eb/software/EasyBuild/3.3.0/lib/python2.7/site-packages/vsc_base-2.5.8-py2.7.egg/vsc/utils/rest.py", line 206, in get_connection
    connection = self.opener.open(request)
  File "/usr/lib64/python2.7/urllib2.py", line 435, in open
    response = meth(req, response)
  File "/usr/lib64/python2.7/urllib2.py", line 548, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib64/python2.7/urllib2.py", line 467, in error
    result = self._call_chain(*args)
  File "/usr/lib64/python2.7/urllib2.py", line 407, in _call_chain
    result = func(*args)
  File "/usr/lib64/python2.7/urllib2.py", line 633, in http_error_302
    new = self.redirect_request(req, fp, code, msg, headers, newurl)
  File "/usr/lib64/python2.7/urllib2.py", line 594, in redirect_request
    raise HTTPError(req.get_full_url(), code, msg, headers, fp)
urllib2.HTTPError: HTTP Error 307: Temporary Redirect
```

**Comment by zao** [comment]
The upstream account can be altered with `--pr-target-account=easybuilders` as a workaround.

**Comment by boegel** [comment]
Should be fixed by #2258, let's follow up there

Thanks for reporting this @zao!
